asv_bench/benchmarks/frame_methods.py

```diff
@@ -609,4 +609,15 @@ def time_dataframe_describe(self):
         self.df.describe()
 
 
+class SelectDtypes:
+    params = [10, 100, 1000, 10000, 100000]
```

Comment:
Can you post ASV results? Might not need all of these parameters, especially if they take significant time to run

Reply:
check my comment above https://github.com/pandas-dev/pandas/pull/28447#issuecomment-531514169
If you have a suggestion on how to extract the results in a more readable/compact way, I'll change the comment